packages/ui-app/src/Input.js

```diff
@@ -86,6 +88,7 @@ export default class Input extends React.PureComponent<Props, State> {
                 : 'off'
             }
           />
+          {isEditable ? <i className='edit icon' /> : null}
```

Comment:
We are specifying the icon in 2 places, i.e. classname and then as an actual icon. Shouldn't 1 place be sufficient? (And yes, have seen some weird things in this regard before, just feels like we are missing something and it should only be needed once, possibly as this version since we are manually including the `input`)

Reply:
Previously I'd tried loads of different combinations trying to get the edit icon to be contained within the input field. Unfortunately the way I've done it is the only way I could get it to work. I just tried removing some of the duplicate classes as you suggested and was impressed to find that it still works.
Another approach that I tried previously was creating an `EditIcon` component and passing it down to the `icon` prop of the `Input` component, but I couldn't get that to work.

Reply:
Yes, I've seen funnies with icons + inputs before as well. (I think there is something basic I'm missing.)